acrobat/scripts/contentSecurityPolicy/dev.js

```diff
@@ -10,6 +10,7 @@ const connectSrc = [
   '*.clarity.ms',
   '*.clicktale.net/',
   '*.mktoresp.com',
+  '*.demandbase.com',
```

Comment:
do we need the wildcard here?  for security purposes, it's better not to

Reply:
Updated to `tag-logger.demandbase.com`